tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -95,6 +89,26 @@ enum class TrtTestMode {
   kDynamicShape = 2
 };
 
+string DebugString(const TrtTestMode mode){
```

Comment:
Need a space toward to the end: s/){/) {/

Reply:
Fixed